resources/lib/tubecast/youtube/app.py

```diff
@@ -48,7 +48,8 @@ def has_playlist(self):  # type: () -> bool
         return bool(self.playlist)
 
     def handle_set_playlist(self, data):
-        self.ctt = data["ctt"]
+        if 'ctt' in data:
```

Comment:
maybe we should use data.get(key, default) more instead of guarding every property access?

Reply:
Indeed, this might be safer.

I have no idea what is this `ctt` key. But for some context, I am not logged in to any Google account on the Android device so maybe this `ctt` flag reflects that?